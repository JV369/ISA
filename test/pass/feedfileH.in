./feedreader -f test/pass/feedfileH.txt
