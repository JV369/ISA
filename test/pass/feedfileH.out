*** Test ***
Test entry
Test entry 2

*** Test RDF ***
Test item RDF
Test item RDF2

*** TestOfRSS ***
Test RSS2
Test RSS1
